build-support/bin/release.sh

```diff
@@ -100,8 +100,7 @@ RELEASE_PACKAGES=(
 function execute_packaged_pants_with_internal_backends() {
   pip install --ignore-installed \
     -r pants-plugins/3rdparty/python/requirements.txt &> /dev/null && \
-  pants \
-    --python-repos-repos="${DEPLOY_WHEEL_DIR}" \
+  PANTS_PYTHON_REPOS_REPOS="${DEPLOY_WHEEL_DIR}" pants \
```

Comment:
Things used to be this way, but were changed and "working" only because the change was when the pants version was `1.4.0.dev14` which was already released to pypi; thus distributions of pants were being resolved from there. The explanation for why an env var is needed is here: https://github.com/pantsbuild/pants/blob/master/src/python/pants/init/plugin_resolver.py#L133-L136